pkg/core/resources/model/resource.go

```diff
@@ -30,6 +30,7 @@ type Resource interface {
 	GetSpec() ResourceSpec
 	SetSpec(ResourceSpec) error
 	Validate() error
+	Meshed() bool
```

Comment:
Can we do
```
type ResourceScope string

const (
	ScopeMesh = "Mesh"
	ScopeGlobal = "Global"
)

Scope() ResourceScope
```
(I'm open to better name than "Global"), but I want this to be "enum" so we have a room to extend this in the future if needed.